transmart-schemas/src/main/resources/db/changelog/changes/20191907041159_biomart_user_views_with_bitset.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<!--
+  ~ Copyright (c) 2019  The Hyve B.V.
+  ~  This file is distributed under the GNU General Public License
+  ~  (see accompanying file LICENSE).
+  -->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+    <preConditions onFail="WARN">
+        <dbms type="PostgreSQL" />
```

Comment:
Now only the precondition is checked for `postgresql`, that's why `gradle check` fails, as it uses `h2`. The `dbms` attribute should probably be on the change set.

Reply:
You can put all view related changes in a single change set with `dbms="postgresql"` and have the precondition in there. The creation of the `biomart_user` schema can be in a separate change set.

Reply:
diveded changesets to 2 files. one as basic without pg_bitcount extension, the other one with it.

Reply:
not sure if it's useful to create them separately. Looks like `patient_num_boundaries` view is used only by views that require  pg_bitcount extension, so without it there is no use for this view.